master/usage/decommissioning-a-node.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Decommissioning a Node
+---
+
+### Why you might be interested in this guide
+
+- You are decomissioning a host running calico/node or removing it from your
+  cluster.
+- You are renaming a Node.
+- You are receiving an error about an IP address already in use.
+- Hosts are regularly added and removed from your cluster.
+
+### Purpose of this page
+
+Provide guidance on how to remove a host that is part of a Calico cluster
+and clean up the associated [Node resource][Node resource reference]
+information.
+
+### Prerequisites
+
+- Prior to removing any Node resource from the datastore the calico/node
+  container should be stopped on the corresponding host and it should be
+  ensured that it will not be restarted.
+- You must have [calicoctl configured][calicoctl setup] and operational to run
+  the commands listed here.
+
+### Removing a Calico Node resource
+
+**Note:**
+Removing a Node resource will also remove the Workload Endpoint, Host
+Endpoint, and IP Address resources and any other sub configuration items
+associated with that Node.
+
+**Warning**
+- Before running the following make sure you are certain that the node is no
+  longer in service as all workloads associated with the node will be removed
+  from the datastore.
+- Any configuration specific to the node will be removed.  This would be
+  configuration like node BGP peerings or custom Felix configs.
+
+### Removing a single Calico Node resource
+
+See the example below for how to remove a node with the calicoctl command.
+
+**Caution** See the [Warning](#removing-a-calico-node-resource) above
+
+```
+calicoctl delete node <nodeName>
+```
+
+### Removing multiple Calico Node resources
```

Comment:
I'm not too fussed - I tend to think it's probably not necessary, but it doesn't seem to hurt.